libraries/chain/webassembly/crypto.cpp

```diff
@@ -159,10 +169,20 @@ namespace eosio { namespace chain { namespace webassembly {
                               span<const char> modulus,
                               span<char> out) const {
       if (context.control.is_producing_block()) {
-         static constexpr int byte_size_limit = 256; // Allow up to 2048-bit values for base, exp, and modulus.
+         unsigned int base_modulus_size = std::max(base.size(), modulus.size());
+
+         if (base_modulus_size < exp.size()) {
+            EOS_THROW(subjective_block_production_exception, 
+                      "mod_exp restriction: exponent bit size cannot exceed bit size of either base or modulus");
+         }
+
+         static constexpr uint64_t bit_calc_limit = 101;
```

Comment:
Where did 101 come from? I think we still want to support 2048 bit exponent if base or modulus is still max 2048 bits? In that case it would equal 104.

Reply:
If exponent is small enough (which usually is for RSA crypto) 2048+ bit modulus can be used. 
e.g standard RSA exponent 0x010001 with 2048 bit modulus gives 74 bits with above formula.

Reply:
Ok yeah you're right, for backward compatibility with previous semantics  constraint should be `104` to get in all 3 ints at 2048 bits.

Reply:
I updated it from 101 to 106. See the fc submodule comments for reasoning.